Dockerfile.staging

```diff
@@ -0,0 +1,14 @@
+FROM nginx:latest
+EXPOSE 80
+
+# Configure Nginx for http basic auth
+COPY nginx/staging-nginx.conf /etc/nginx/nginx.conf
+COPY nginx/http-basic-auth-start.sh /etc/nginx/http-basic-auth-start.sh
+
+# Add website contents
+RUN echo "hello world!" > /usr/share/nginx/html/index.html
+
+# Prevent web scrapers
+RUN echo "User-agent: * Disallow: /" > /usr/share/nginx/html/robots.txt
+
+CMD /etc/nginx/http-basic-auth-start.sh
```

Comment:
Curious, is this being run as root, or does the nginx docker image provide an nginx user?

Reply:
I think there's an nginx user
https://github.com/nginxinc/docker-nginx/blob/master/Dockerfile-debian.template#L10
I suspect this is the Dockerfile in use.